ml-agents/mlagents/trainers/rl_trainer.py

```diff
@@ -46,23 +47,19 @@ def end_episode(self) -> None:
                 rewards[agent_id] = 0
 
     def _update_end_episode_stats(self, agent_id: str) -> None:
-        self.stats["Environment/Episode Length"].append(
-            self.episode_steps.get(agent_id, 0)
-        )
         self.episode_steps[agent_id] = 0
         for name, rewards in self.collected_rewards.items():
             if name == "environment":
                 self.cumulative_returns_since_policy_update.append(
                     rewards.get(agent_id, 0)
                 )
-                self.stats["Environment/Cumulative Reward"].append(
-                    rewards.get(agent_id, 0)
-                )
                 self.reward_buffer.appendleft(rewards.get(agent_id, 0))
                 rewards[agent_id] = 0
             else:
-                self.stats[self.policy.reward_signals[name].stat_name].append(
-                    rewards.get(agent_id, 0)
+                stats.stats_reporter.add_stat(
+                    self.summary_path,
```

Comment:
Feels a little weird that the "category" here is a filepath.  The "category" seems like something that should just be the filename / behavior name, whereas the base bath should be something used to configure the Tensorboard writer.

Reply:
Seems to me like the summary_path is exactly that - just the name of the behavior and not the full path